tests/runner/highlighter.test.ts

```diff
@@ -18,20 +18,90 @@
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
 import * as ts from "typescript";
-import { SyntaxHighlighter } from "../../src/runner/highlighter";
+import { HighlightedToken, SonarTypeOfText, SyntaxHighlighter } from "../../src/runner/highlighter";
 import { parseString } from "../../src/utils/parser";
 
-it("should highlight 'this'", () => {
-  const input = { file_content: "class A {\n a: string;\n getA() { return this.a; }\n}" };
+it("should highlight keywords", () => {
+  const result = actual(
+    `class A {
+     get b() {
+       return this.a;
+     }
+     static foo() {
+       if (cond);
+     }
+     a: string;
+  }`,
+  );
+  expect(result).toContainEqual(token(1, 0, 1, 5, "keyword")); // class
```

Comment:
@stas-vilchik As discussed, I prefer these assertions to be explicit in test code.